src/routes/search.py

```diff
@@ -25,27 +26,31 @@ def get_blueprint():
 
 
 @search_blueprint.route("/search_ingredients", methods=["POST"])
-def search_ingredients():
-    if flask.request.method == "POST":
-        ingredients = flask.request.form.get("searchIngredients")
+def searchByingredients():
+    if request.method == "POST":
+        ingredients = request.form.get("searchIngredients")
         # prints what the user searches for to the terminal
         print(ingredients)
-        returnedDict = api.spoonacular.search_ingredients(ingredients)
+        returnedDict = search_ingredients(ingredients)
         # prints the 10 recipes to the terminal
         print(returnedDict)
```

Comment:
Make sure the print statements are gone once the method is used as they will not be needed